src/View.Uno.WinUI/Internal/MainDispatcher/MainDispatcherScheduler.Windows.cs

```diff
@@ -0,0 +1,40 @@
+﻿#if WINDOWS10_0_18362_0
+// Implement an CoreDispatcherScheduler equivalence for Windows .NET 6 https://github.com/nventive/Nventive.View/issues/50
```

Comment:
I don't understand. Is it working?

Reply:
I'll add a _TODO_ in front of those comments, but those are related to enhancement issues that need to be implemented in the future. The issues have more details about each specific cases

Reply:
So in the future because the current features still work?